src/annotator/components/ContentInfoBanner.js

```diff
@@ -23,13 +24,18 @@ import {
  *   @param {ContentInfoConfig} props.info
  */
 export default function ContentInfoBanner({ info }) {
+  // Format item title to show subtitle
+  let itemTitle = info.item.title;
+  if (info.item.subtitle) {
+    itemTitle += `: ${info.item.subtitle}`;
+  }
   return (
     <div
       className={classnames(
         'h-10 bg-white px-4 text-slate-7 text-annotator-base border-b',
         'grid items-center',
         // Two columns in narrower viewports; three in wider
-        'grid-cols-[100px_minmax(0,auto)] gap-x-4',
+        'grid-cols-[100px_minmax(0,auto)]',
```

Comment:
This little change (removal of a gap) is in a separate commit and is a remedy for a small layout inconsistency when in narrower viewports.